packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -517,12 +517,18 @@ class CheckboxListTile extends StatelessWidget {
     Widget? leading, trailing;
     switch (controlAffinity) {
       case ListTileControlAffinity.leading:
-        leading = control;
+        leading = Focus(
+            descendantsAreFocusable: false,
```

Comment:
Instead of wrapping this here, you could do it once before the switch:

```dart
 control = Focus(
            descendantsAreFocusable: false,
            canRequestFocus: false,
            child: control);
```